src/context/filter-context.js

```diff
@@ -0,0 +1,34 @@
+import { createContext, useContext, useReducer } from "react";
+import { filterReducer } from "../reducers";
+
+const filterContext = createContext();
```

Comment:
Convention followed is that the created context object is written in PascalCase. Maybe you could adopt it.